### DISCUSSION TITLE
Customisable Conduit Range + related

### DISCUSSION THREAD

**Discussion by kevindwitama** [PurpurMC/Purpur#559] [__Completed__] [answered]
The beacon range is already customisable, shouldn't the conduit's be as well? Here are my suggestions:

- The range for conduit power increases every seven blocks we add in the frame. By default, that range is 16. I think there should be a config to change how much the range increases.
- A complete frame also deals damage to hostile mobs in water or rain within 8 blocks by default, which is pretty measly. I think we should be change that range as well.
- By default, only full prismarine type blocks count to activate a conduit, not slabs or double slabs. If possible, there should be a true/false config to include slabs as well.

The config options might look something like this, along with default values:

```
conduit:
  effect-range-per-level: 16
  damage-range: 8
  allow-slabs-in-frame: false
```

**Comment by encode42** [comment, marked as answer]
Closed in https://github.com/pl3xgaming/Purpur/commit/bd98fb59c6ce77ee2f1db842cfc2b6f8445ff49a
